app/controllers/steps/appeal/case_type_controller.rb

```diff
@@ -11,5 +11,22 @@ def edit
     def update
       update_and_advance(CaseTypeForm)
     end
+
+    private
+
```

Comment:
Should we move these methods to an abstract "FirstStepController" class? Then, whichever step is actually the first one can inherit from that, instead of inheriting from StepController. Just a thought.